### PR TITLE
Add blog post: LibrePCB Meets FontoBene

### PR DESCRIPTION
**Pull request by ubruhin**

### CODE REVIEW

File: content/blog/2018-04-19_librepcb_meets_fontobene.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### Why a stroke font instead of OTF/TTF?
+
+It may sound strange to implement a primitive stroke font in 2018 when we
+actually have much more powerful TrueType and OpenType fonts since many years.
+But there are still reasons to stick with the good old stroke fonts.
+
+One reason is that OpenType and TrueType fonts are very complex and thus require
+high effort to integrate them into LibrePCB. For example the Gerber export is
+not as trivial as you may expect because Gerber doesn't support fonts. On the
```
Comment (dbrgn) [inline content/blog/2018-04-19_librepcb_meets_fontobene.md]: What about replacing "fonts" with "Bézier curves"?

File: content/blog/2018-04-19_librepcb_meets_fontobene.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Beside their simplicity, stroke fonts even have a clear advantage compared to
+OTF/TTF. Since you can set the stroke width to a value greater than or equal to
+the minimum silkscreen line width specified by your PCB manufacturer, you can be
+sure that the text items are printed properly and thus are perfect readable.
+
+[![Text Properties](/img/board_editor_text_properties.png)](/img/board_editor_text_properties.png)
+
+### Introducing FontoBene
+
+Unfortunately we were not happy with the available stroke fonts because of
```
Comment (dbrgn) [inline content/blog/2018-04-19_librepcb_meets_fontobene.md]: fonts -> font formats?

File: content/blog/2018-04-19_librepcb_meets_fontobene.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+several reasons. [Hershey](https://en.wikipedia.org/wiki/Hershey_fonts) and
+[NewStroke](https://vovanium.ru/sledy/newstroke/en) have many edges because they
+only consist of straight line segments. LFF from LibreCAD and CXF from QCad
+look much smoother because they support circular arc segments, but unfortunately
+there aren't any clear specifications available and the existing fonts have
+non-permissive, partly even unknown licenses.
+
+![FontoBene](/img/fontobene.png)
```
Comment (dbrgn) [inline content/blog/2018-04-19_librepcb_meets_fontobene.md]: Hm, you're inserting this screenshot just after the section where you complain about missing arcs, but the example font clearly consists of only straight lines :)

File: content/blog/2018-04-19_librepcb_meets_fontobene.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+[clear specifications](https://github.com/fontobene/fontobene/blob/master/SPECIFICATION.md)
+for a new stroke font file format, called [FontoBene](https://github.com/fontobene/fontobene/).
+Because the project is completely independent from any software project (even
+from LibrePCB), our hope is that it will also be used by other projects some time.
+
+### Outlook
+
+Currently LibrePCB uses the [NewStroke](https://vovanium.ru/sledy/newstroke/en)
+font which we converted to FontoBene, mainly because it is available under the
+CC0 Public Domain license. So we don't yet use the advantages of having support
```
Comment (dbrgn) [inline content/blog/2018-04-19_librepcb_meets_fontobene.md]: So we don't yet use the advantages -> This means that we currently don't yet take advantage

**Comment by dbrgn** [comment]
@ubruhin if you use "commit --fixup" instead of amending, then it's easier to see the changes in the pull request history :) you can always squash before / when merging.

File: content/blog/2018-04-19_librepcb_meets_fontobene.md (hunk 3 above)
Comment (dbrgn) [reply content/blog/2018-04-19_librepcb_meets_fontobene.md]: Yeah, that's probably the easiest solution. Maybe between the two paragraphs in the outlook section?

**Comment by ubruhin** [comment]
@dbrgn I know, sorry for not doing it ;) I rebased this branch onto master because I fixed the broken CI first, and then I squashed the two commits I've made on this branch...

Now I replaced the FontoBene image with a font which uses arcs, so now one can see how it could look like. In addition, I added a new chapter about embedding fonts into projects because I think this could be an interesting information for some users.

**Comment by ubruhin** [comment]
@dbrgn ready to publish?

**Pull request merged**